resources/js/Components/Form/DmEntryCreateForm.tsx

```diff
@@ -263,9 +262,13 @@ const DmEntryCreateForm = ({
         <StyledGrid container spacing={4}>
             <Grid item xs={4}>
                 {type === 'Create' ? (
-                    <Link href={route('dm-entry.index')}>
-                        <Button fullWidth>{t('common.cancel')}</Button>
-                    </Link>
+                    <Button
```

Comment:
do a quick refactor here please, use ternary expression for the correct onClick and the text to be displayed